src/Web/.editorconfig

```diff
@@ -10,9 +10,15 @@ insert_final_newline = true
 
 # Matches multiple files with brace expansion notation
 # Set default charset
-[*.{js,tsx,ts,scss}]
+[*.{js,tsx,ts}]
 charset = utf-8
 indent_style = tab
+indent_size = 2
```

Comment:
Proponuję, tak jak sie umawialismy, zrobic jedynie indent_style w .editorconfig, natomiast sama wielkosc juz w ustawieniach VS Code. Bo potem  lądują zmiany np. w package.json polegające na zmianie wielkosci wcięć.

Reply:
To się nie nadpisuje, bo to jest dalej jako tab ale mi się wyświetla jako 2 spacje, a tobie jako 4, w kodzie jest ten sam symbol, dodałem do gitignora żeby się nie nadpisywał